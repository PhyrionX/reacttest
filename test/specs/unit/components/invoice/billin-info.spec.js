import React from 'react';
import { render } from '@testing-library/react'
import BillinInfo from '../../../../../src/components/invoice/billin-info';

describe('<BillinInfo />', () => {
  it('should mount empty component', () => {
    const { asFragment } = render(<BillinInfo />);
    
    expect(asFragment()).toMatchSnapshot();
  })

  it('should mount with items', () => {
    const { asFragment } = render(<BillinInfo items={[
      {
        title: 'testTitle',
        info: 'testInfo'
      }
    ]}/>);
    
    expect(asFragment()).toMatchSnapshot();
  })
})
