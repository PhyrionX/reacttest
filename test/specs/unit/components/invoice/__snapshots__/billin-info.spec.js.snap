// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BillinInfo /> should mount empty component 1`] = `
<DocumentFragment>
  <div
    class="billin-info"
  />
</DocumentFragment>
`;

exports[`<BillinInfo /> should mount with items 1`] = `
<DocumentFragment>
  <div
    class="billin-info"
  >
    <div
      class="billin-info__item"
    >
      <p
        class="billin-info__title"
      >
        testTitle
      </p>
      <p
        class="billin-info__info"
      >
        testInfo
      </p>
    </div>
  </div>
</DocumentFragment>
`;
